website/blog/2021-11-18-2.5.0.md

```diff
@@ -0,0 +1,366 @@
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+function foo() { var bar = 1; delete bar; }
+
+// Prettier 2.4
+SyntaxError: Deleting local variable in strict mode. (1:31)
+> 1 | function foo() { var bar = 1; delete bar; }
+    |                               ^
+
+// Prettier 2.5
+function foo() {
+  var bar = 1;
+  delete bar;
+}
+```
+
+#### Respect spacing for between expressions and parentheses in embedded CSS ([#11800](https://github.com/prettier/prettier/pull/11800) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+<!-- Optional description if it makes sense. -->
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+const paragraph2 = css`
+  transform: ${expr}(30px);
+`;
+
+// Prettier 2.4
+const paragraph2 = css`
+  transform: ${expr} (30px);
+`;
+
+// Prettier 2.5
+const paragraph2 = css`
+  transform: ${expr}(30px);
+`;
+
+```
+
+### TypeScript
+
+#### Avoid extra offset in arrow function body when using long types ([#11515](https://github.com/prettier/prettier/pull/11515) by [@kachkaev](https://github.com/kachkaev) and [@thorn0](https://github.com/thorn0))
+
+Starting with Prettier 2.3.0, type declarations in arrow functions could affect function body offset. Changing the length of the type annotation could produce large diffs and thus increased the chance of git conflicts. To prevent this, function body offset was stabilized.
+
+<!-- prettier-ignore -->
+```tsx
+// Input
+const MyComponentWithLongName: React.VoidFunctionComponent<MyComponentWithLongNameProps> = ({ x, y }) => {
+  const a = useA();
+  return <div>{x + y + a}</div>;
+};
+
+// Prettier 2.2 and below
+const MyComponentWithLongName: React.VoidFunctionComponent<MyComponentWithLongNameProps> = ({
+  x,
+  y,
+}) => {
+  const a = useA();
+  return <div>{x + y + a}</div>;
+};
+
+// Prettier 2.4
+const MyComponentWithLongName: React.VoidFunctionComponent<MyComponentWithLongNameProps> =
+  ({ x, y }) => {
+    const a = useA();
+    return <div>{x + y + a}</div>;
+  };
+
+// Prettier 2.5
+const MyComponentWithLongName: React.VoidFunctionComponent<
+  MyComponentWithLongNameProps
+> = ({ x, y }) => {
+  const a = useA();
+  return <div>{x + y + a}</div>;
+};
+```
+
+#### Fix formatting for tagged template decorators ([#11717](https://github.com/prettier/prettier/pull/11717) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+<!-- prettier-ignore -->
+```tsx
+// Input
+class Test {
+  @foo`bar`
+  test: string = "test"
+}
+
+// Prettier 2.4
+class Test {
+  @(foo`bar`)
+  test: string = "test"
+}
+
+// Prettier 2.5
+class Test {
+  @foo`bar`
+  test: string = "test"
+}
+```
+
+### SCSS
+
+#### Fix wildcard syntax in `@forward` ([#11482](https://github.com/prettier/prettier/pull/11482)) ([#11487](https://github.com/prettier/prettier/pull/11487) by [@niksy](https://github.com/niksy))
```

Comment:
#11482 is an issue, not a pull request. How about removing this link from [`changelog_unreleased/scss/11487.md`](https://github.com/prettier/prettier/blob/59b5eb4e1faf912dd46759c98b45cfe5807e7a9f/changelog_unreleased/scss/11487.md) to make all header structures consistent?

```diff
- #### Fix wildcard syntax in `@forward` (#11482) (#11487 by @niksy)
+ #### Fix wildcard syntax in `@forward` (#11487 by @niksy)
```

Reply:
Hmm, odd. Looks like this fix was already mentioned in [2.4.1 CHANGELOG](https://github.com/prettier/prettier/blob/main/CHANGELOG.md#241) 🤔

Reply:
Thank you. I forgot clean `changelog_unreleased` after releasing 2.4.1..